api-reference/beta/resources/group.md

```diff
@@ -77,6 +77,7 @@ by providing a [delta](../api/group_delta.md) function.
 |membershipRule|String|The rule that determines members for this group if the group is a dynamic group (groupTypes contains "**DynamicMembership**"). For more information about the syntax of the membership rule, please refer to [Membership Rules syntax](https://azure.microsoft.com/en-us/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/)|
 |membershipRuleProcessingState|String|Indicates whether the dynamic membership processing is on or paused. Possible values are "On" or "Paused"|
 |onPremisesLastSyncDateTime|DateTimeOffset|Indicates the last time at which the object was synced with the on-premises directory.The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`. Read-only. Supports $filter.|
+|onPremisesProvisioningErrors|[onPremisesProvisioningError](onpremisesprovisioningerror.md) collection|Directory sync errors during provisioning. |
```

Comment:
I thought that syncing (via AD Connect) when you have inconsistent or conflicting properties is when you get sync errors.  The errors are generally due to on-premises properties - so yes not *caused* by AD Connect, but are sync errors when using the AD Connect appliance.  It's weird to call this "Directory sync errors" IMHO